src/env.ts

```diff
@@ -4,8 +4,11 @@ import { DefaultNetworkConfigurations } from "@leather.io/models";
 export const env = {
   BITCOIND_URL: process.env.BITCOIND_URL || "http://localhost:18443",
   EMILY_URL: process.env.EMILY_URL,
-  MEMPOOL_API_URL:
-    process.env.MEMPOOL_API_URL || "http://localhost:8083/api/v1/",
+  MEMPOOL_API_URL: (
+    process.env.MEMPOOL_API_URL || "http://localhost:8083/api"
+  ).replace(/\/$/, ""),
+  MEMPOOL_URL: process.env.MEMPOOL_URL,
```

Comment:
Can we do the same trailing slash replace also for `MEMPOOL_URL`?

Reply:
smart, fixed!